examples/MMF-activity-recognition/README.md

```diff
@@ -0,0 +1,150 @@
+### MultiModal (MMF) Framework
```

Comment:
Some this is outdated. I think in general it might make more sense to point to MMF website rather than going in a deep dive here as MMF website would always be the most up to date source.